tests/foreman/ui/test_contentviews.py

```diff
@@ -1255,14 +1253,14 @@ def test_delete_version_non_default(self):
             name=gen_string('alpha'),
             organization=org,
         ).create()
-        cv.set_repository_ids([repo.id])
+        cv.set_repository_ids({u'repository_ids': [repo.id]})
         cv.publish()
 
         cv_info = cv.read_json()['versions'][0]
         version = 'Version {0}'.format(cv_info['version'])
         cvv = entities.ContentViewVersion(id=cv_info['id'])
         lc_env = entities.LifecycleEnvironment(organization=org).create()
-        cvv.promote(lc_env.id)
+        cvv.promote({'environment_id': lc_env.id})
```

Comment:
Not a blocker but you may want to always make your strings `unicode` like some of your other changes.